tests/unit/configuration_subsystem/test_invalid_params.py

```diff
@@ -141,15 +141,3 @@ def test(subcommand, param, look_for):
     else:
         # argparse choice error
         test(subcommand, "", "choose from")
-
```

Comment:
this test can stay, but now just for collection, in fact, images needs to be added here

Reply:
+1 I should not have removed the entire test case. :-)